src/components/Main/SearchArtistPage/SearchBar/SearchBar.js

```diff
@@ -0,0 +1,33 @@
+import { useState } from "react";
+
+function SearchBar({ onSearch }) {
+  const [searchTerm, setSearchTerm] = useState("");
+
+  const handleInputChange = (e) => {
+    setSearchTerm(e.target.value);
+  };
+
+  const handleSearch = () => {
+    onSearch(searchTerm);
+  };
+
+  return (
+    <div className="flex items-center space-x-4 mb-4 justify-between ">
+      <input
```

Comment:
If you have `inpust` then they should be inside tag `form` to be able to submit the form